test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/002_ATF_P_TC_existing_navi_app_no_certificate.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] SDL must start PTU for navi app right after app successfully registration
+--
+-- Description:
+-- In case navigation app connects and sucessfully registers on SDL (opens RPC 7 service)
+-- and PolicyTable has NO "certificate" at "module_config" section of LocalPolicyTable
+-- SDL must start PolicyTableUpdate process on sending SDL.OnStatusUpdate(UPDATE_NEEDED) to HMI to get "certificate"
+--
+-- 1. Used preconditions:
+-- Navi app exists in LP, no certificate in module_config
+--
+-- 2. Performed steps
+-- Register navi application.
+--
+-- Expected result:
+-- Application is registered successfully.
+-- SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local mobile_session = require('mobile_session')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+
+--[[ General Precondition before ATF start ]]
+testCasesForPolicyCeritificates.update_preloaded_pt(config.application1.registerAppInterfaceParams.appID, false)
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_connectMobile()
+  self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_RAI_PTU_Trigger()
+  local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+  EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2)
```

Comment:
@dboltovskyi, according to [PTU-Proprietary] Send BC.PolicyUpdate to HMI in case PTU is triggered, the time is not specified and usually SDL is sending immediately after snapshot creation. But to be sure I will add additional delay for 2 notifications (2 x 10sec). See 7850c48712ff956897e98d4b6f159d6e39eeba2b